.rubocop.yml

```diff
@@ -76,8 +76,8 @@ RSpec/AnyInstance:
   Enabled: false
 
 Metrics/CyclomaticComplexity:
-  Max: 14
+  Max: 17
```

Comment:
What is causing this? The new logic in PermissionChecker I assume?

Reply:
Yeah, `CyclomaticComplexity` is the number of possible independent paths that the method could take

Reply:
Maybe that is too high,
That may have a downside on our code quality and make refactoring more challenging,
No strong opinion tho,